Neo4j dashboards in Aura (#581)
neo4j/docs-aura: modules/ROOT/pages/dashboards/visualizations/index.adoc

= Visualizations
:description: The different visualization options in Neo4j dashboards.

A visualization is embedded in a dashboard card.
Visualizations have different types, each of which expect different types of data.

== Writing Cypher queries

Each visualization uses a Cypher query specified in the dashboard card to retrieve data from your Neo4j database and display it.

> Edit the query associated with a dashboard card by clicking the three dots icon at the top right of a card and then **Edit card**.

Reviewer: ```suggestion
Edit the query associated with a dashboard card using the more menu *[...]* at the top right of a card and then **Edit card**.
```

= Visualizations
:description: The different visualization options in Neo4j dashboards.

A visualization is embedded in a dashboard card.
Visualizations have different types, each of which expect different types of data.

== Writing Cypher queries

Each visualization uses a Cypher query specified in the dashboard card to retrieve data from your Neo4j database and display it.
Edit the query associated with a dashboard card by clicking the three dots icon at the top right of a card and then **Edit card**.

> Any Cypher syntax is supported, including link:https://neo4j.com/developer/neo4j-apoc/[APOC], link:https://neo4j.com/docs/graph-data-science/current/[GDS], and link:https://neo4j.com/docs/operations-manual/current/fabric/queries/[Fabric].

Author: rephrased this entire bit, and added links to the cypher manual and aura + cypher
